### ISSUE TITLE
DietPi-Software | ViFM  will not be marked as installed after installation

### ISSUE THREAD

**Issue by k-plan** [MichaIng/DietPi#373]
- install ViFM via `dietpi-software`

```
root@RPi-3-test:~# vifm -v
Version: 0.7.8
Git commit hash:
Compiled at: Nov  6 2014 07:07:04
```
- after reboot, it will not be marked as installed.
  ![160604-0002](https://cloud.githubusercontent.com/assets/17237601/15800108/4dd603c8-2a71-11e6-90b9-49f3b9ef6226.gif)
- but you can uninstall ViFM via `dietpi-uninstall`
  ![160604-0001](https://cloud.githubusercontent.com/assets/17237601/15800013/1681d458-2a6f-11e6-8c74-4a7456cd48f3.gif)

**Comment by Fourdee** [comment]
@k-plan 

Good spot :+1:  fixed: https://github.com/Fourdee/DietPi/commit/565dc91463f956bc1a1220f5b244007fab6e89ba

**Comment by xenfomation** [comment]
Fast work, sir!  Thanks @k-plan & @Fourdee

**Comment by k-plan** [comment]
@Fourdee 

> Good spot 👍 fixed: 565dc91

Testing:

```
 ───────────────────────────────────────
 DietPi     | 16:00 | Sun 05/06/16
 ───────────────────────────────────────
 V120       | RPi 3 Model B (armv7l)
 ───────────────────────────────────────
```

![160605-0004](https://cloud.githubusercontent.com/assets/17237601/15805892/0e676e04-2b37-11e6-8386-20795425f6ec.gif)
![160605-0005](https://cloud.githubusercontent.com/assets/17237601/15805896/16fd2a4a-2b37-11e6-8818-5ce5f7966e8b.gif)

Works! 😃  👍

**Comment by Fourdee** [comment]
Thanks @k-plan :+1: 

Marking this as closed :)
